www/nginx/src/opnsense/www/js/nginx/src/controller/LogView.js

```diff
@@ -161,6 +191,19 @@ const LogView = Backbone.View.extend({
         this.page_entry_count = event.target.value;
         this.current_page = 0;
         this.update();
+    },
+
+    toggle_column: function (event) {
+        event.stopPropagation();
+        let uid = this.collection.uuid;
+        let type = this.type;
+        let field = $(event.currentTarget).find('input').prop('value');
+        let ishidden = localStorage.getItem('visibleColumns[' + type + '][' + uid + '][' + field + ']') === 'false';
```

Comment:
seems to be the same in render. Maybe extract it as a shared functionality on class level or as function in the file (I prefer as a method on the class).